src/ralph/models/converter.py

```diff
@@ -24,18 +21,50 @@ class ConversionItem:
     transformers: tuple[Callable[[Any], Any]]
 
     def __init__(self, dest: str, src=None, transformers=lambda _: _):
+        """Initializes ConversionItem.
+
+        Args:
+            dest (str): The destination path where to place the converted value.
+            src (str or None): The source from where the value to convert is fetched.
+                - When `src` is a path (ex. `context__user_id`) - the value is the item
+                    of the event at the path.
+                - When `src` is `*` - the value is the whole event.
+                - When `src` is `**` - the value is the original event string.
+                - When `src` is `None` - the value is `None`.
+            transformers (function or tuple of functions): The function(s) to apply on the source
+                value.
+        """
+
         object.__setattr__(self, "dest", tuple(dest.split(MODEL_PATH_SEPARATOR)))
-        src = tuple(src.split(MODEL_PATH_SEPARATOR)) if src else None
+        special = [None, "*", "**"]
+        src = src if src in special else tuple(src.split(MODEL_PATH_SEPARATOR))
         object.__setattr__(self, "src", src)
         transformers = tuple([transformers]) if callable(transformers) else transformers
         object.__setattr__(self, "transformers", transformers)
 
-    def get_value(self, event: dict):
-        """Returns fetched source value after having applied transformers to it."""
+    def get_value(self, event: dict, event_str: str):
+        """Returns fetched source value after having applied all transformers to it.
+
+        Args:
+            event (dict): The event to convert.
+            event_str (dict): The original event string.
+
+        Raises:
+            ConversionException: When a field transformation fails.
+        """
 
-        value = get_dict_value_from_path(event, self.src)
-        for transformer in self.transformers:
-            value = transformer(value)
+        if self.src == "*":
+            value = event
+        elif self.src == "**":
+            value = event_str
+        else:
+            value = get_dict_value_from_path(event, self.src)
```

Comment:
```suggestion
        if self.raw_input:
            value = data
        else:
            value = get_dict_value_from_path(data, self.src)
```

Reply:
Thanks) updated this part, but now it's the `src` field deciding if we use the whole event or a specific field.